explorer/interpreter/resolve_names.cpp

```diff
@@ -604,9 +604,8 @@ auto NameResolver::ResolveNamesImpl(Statement& statement,
           ResolveNames(if_stmt.condition(), enclosing_scope));
       CARBON_RETURN_IF_ERROR(
           ResolveNames(if_stmt.then_block(), enclosing_scope));
-      if (if_stmt.else_block().has_value()) {
-        CARBON_RETURN_IF_ERROR(
-            ResolveNames(**if_stmt.else_block(), enclosing_scope));
+      if (auto else_block = if_stmt.else_block(); else_block.has_value()) {
```

Comment:
```suggestion
      if (auto else_block = if_stmt.else_block()) {
```

I think the explicit `.has_value()` is redundant, seems cleaner to remove it?

Reply:
done

Reply:
Missing a push? I don't see any changes in the file.

Reply:
Oh sorry I missed it by mistake.